DESCRIPTION

```diff
@@ -40,7 +40,9 @@ Suggests: BiocStyle(>= 1.7.7),
           JASPAR2014(>= 1.4.0),
           knitr(>= 1.11),
           testthat,
-          JASPAR2016(>= 1.0.0)
+          JASPAR2016(>= 1.0.0),
+          JASPAR2018(>= 1.0.0),
+          JASPAR
```

Comment:
The new data package should be JASPAR2020

Reply:
Aziz wanted all the versions in a single package, so I made it that way. I can also make JASPAR2020 separately

Reply:
The thing is you are not supposed to update the data package, i.e., adding stuff to a existing data package.

Reply:
Ok. I will convert it to JASPAR2020 at first convenience sometime during the day.